atomic_reactor/constants.py

```diff
@@ -118,6 +118,10 @@
 DOCKER_BACKOFF_FACTOR = 5
 # docker retries statuses
 DOCKER_CLIENT_STATUS_RETRY = (408, 500, 502, 503, 504)
+# max retries for docker push
+DOCKER_PUSH_MAX_RETRIES = 6
+# how many seconds should wait before another try of docker push
```

Comment:
It is factor which multiplies seconds, I'd not say "how many seconds" in that comments, this comment is valid only for first try.

Reply:
yeah, I just kept the comments consistent with other backoff factors

Reply:
ok